lms/djangoapps/instructor_task/tasks_helper/module_state.py

```diff
@@ -221,6 +225,75 @@ def rescore_problem_module_state(xmodule_instance_args, module_descriptor, stude
 
 
 @outer_atomic
+def override_problem_score_module_state(xmodule_instance_args, module_descriptor, student_module, task_input):
```

Comment:
This function is really long (65 lines).  Do you think there's a good, natural way to break it up into 2 or 3 smaller functions?